### PR TITLE
Fix special characters in PDF version of User Guide

### PR DESCRIPTION
**Pull request by halitanildonmez**
## Overview

Hello! This commit solves the linked issue. Though here are some things that I wanted to point out. 

I had to point to the fonts directory at the `documentation.gradle.kts` file because I was getting errors saying that 
**GEM_FONTS_DIR** was not found. 

After update, I had to download the default fonts from https://github.com/asciidoctor/asciidoctor-pdf/tree/master/data/fonts  

I also changed the "rare"  Japanese charset, U+FF0 emoji. 

Here: https://github.com/asciidoctor/asciidoctor-pdf/blob/v1.5.x/docs/theming-guide.adoc#applying-your-theme it says that one can do `pdf-fontsdir="path/to/fonts;GEM_FONTS_DIR"` yet it did not worked. 

Yet this commit solves the emoji problem.

---

I hereby agree to the terms of the [JUnit Contributor License Agreement](https://github.com/junit-team/junit5/blob/002a0052926ddee57cf90580fa49bc37e5a72427/CONTRIBUTING.md#junit-contributor-license-agreement).

---

### Definition of Done

- [ ] There are no TODOs left in the code
- [ ] Method [preconditions](https://junit.org/junit5/docs/snapshot/api/org.junit.platform.commons/org/junit/platform/commons/util/Preconditions.html) are checked and documented in the method's Javadoc
- [ ] [Coding conventions](https://github.com/junit-team/junit5/blob/HEAD/CONTRIBUTING.md#coding-conventions) (e.g. for logging) have been followed
- [ ] Change is covered by [automated tests](https://github.com/junit-team/junit5/blob/HEAD/CONTRIBUTING.md#tests) including corner cases, errors, and exception handling
- [ ] Public API has [Javadoc](https://github.com/junit-team/junit5/blob/HEAD/CONTRIBUTING.md#javadoc) and [`@API` annotations](https://apiguardian-team.github.io/apiguardian/docs/current/api/org/apiguardian/api/API.html)
- [ ] Change is documented in the [User Guide](https://junit.org/junit5/docs/snapshot/user-guide/) and [Release Notes](https://junit.org/junit5/docs/snapshot/user-guide/#release-notes)
- [ ] All [continuous integration builds](https://github.com/junit-team/junit5#continuous-integration-builds) pass

### CODE REVIEW

**Comment by marcphilipp** [comment]
Thanks! I'd like to avoid copying the fonts. Let's see if we can get @mojavelinux to chime in on https://github.com/junit-team/junit5/issues/2315#issuecomment-664566872. 🙇‍♂️

**Comment by mojavelinux** [comment]
You definitely should not have to copy the fonts. I'll have a look and see if I can detect what is going wrong.

**Comment by halitanildonmez** [comment]
I left a comment at https://github.com/junit-team/junit5/issues/2315#issuecomment-667500931 and the issue should be fixed now but thank you @marcphilipp and @mojavelinux again for the help 😄

File: documentation/documentation.gradle.kts [hunk 1]
```diff
@@ -56,6 +57,7 @@ dependencies {
 asciidoctorj {
 	modules {
 		diagram.use()
+		pdf.version("1.5.3")
```
Comment (marcphilipp) [inline documentation/documentation.gradle.kts:60]: Please define this version as `asciidoctor-pdf.version=1.5.3` in `gradle.properties` and reference it as follows here.

```suggestion
		pdf.version(versions["asciidoctor-pdf"])
```

**Comment by halitanildonmez** [comment]
No problem 😄 Hope it is better now ?

File: documentation/documentation.gradle.kts (hunk 1 above)
Comment (halitanildonmez) [reply documentation/documentation.gradle.kts]: No problem 😄 Hope it is better now ?

File: documentation/documentation.gradle.kts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import org.asciidoctor.gradle.jvm.AbstractAsciidoctorTask
+import org.jetbrains.kotlin.fir.resolve.dfa.contracts.createArgumentsMapping
```
Comment (marcphilipp) [inline documentation/documentation.gradle.kts:2]: :question: This import is unused, isn't it?

Comment (halitanildonmez) [reply documentation/documentation.gradle.kts]: Yeah. Just checked

**Pull request merged**